### PR TITLE
OSDOCS-2883: add balancing algorithm known issue

### PR DESCRIPTION
**Pull request by bmcelvee**
https://issues.redhat.com/browse/OSDOCS-2883

Preview: https://deploy-preview-38406--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes.html#ocp-4-9-known-issues

PM (Jira), SME, QE, and peer review acks received.

### CODE REVIEW

**Comment by bmcelvee** [comment]
@frobware and @quarterpin would you mind reviewing this PR, please? Thanks!

**Comment by bmcelvee** [comment]
Added change requested in https://github.com/openshift/openshift-docs/pull/38405.

File: release_notes/ocp-4-9-release-notes.adoc [hunk 1]
```diff
@@ -2028,6 +2028,26 @@ As a workaround, use fully qualified names, such as `registry.redhat.io/ubi8/ubi
...
+You can check the default setting by entering the following command:
++
+[source,terminal]
+----
+$ oc get deployment -n openshift-ingress router-default -o yaml | grep -A 2 ROUTER_LOAD_BALANCE_ALGORITHM
+        - name: ROUTER_LOAD_BALANCE_ALGORITHM
+          value: leastconn
+----
++
+The `random` option is still available, but routes that would benefit from this algorithmic choice must explicitly request that by setting the following annotation on a per-route basis by entering the following command:
```
Comment (skrthomas) [inline release_notes/ocp-4-9-release-notes.adoc:2042]:
```suggestion
The `random` option is still available. However routes that would benefit from this algorithmic choice must explicitly set that option in an annotation on a per-route basis by entering the following command:
```

Comment (skrthomas) [reply release_notes/ocp-4-9-release-notes.adoc:2042]: Same suggestion here as in the 4.8 PR.

**Review by skrthomas — COMMENTED** [review]
Same suggestion as in 4.8 PR.

**Comment by mcurry-rh** [comment]
LGTM

**Comment by frobware** [comment]
> @frobware and @quarterpin would you mind reviewing this PR, please? Thanks!

Apologies for the delay.

/lgtm

**Pull request merged**